plugins/att_cybersecurity_alienvault_otx/help.md

```diff
@@ -223,18 +246,386 @@ Example output:
 
 ### Triggers
 
-This plugin does not contain any triggers.
+_This plugin does not contain any triggers._
 
 ### Custom Output Types
 
-_This plugin does not contain any custom output types._
+#### _id
```

Comment:
Can we please reformat this custom type name not to start with an underscore `_`? We would ideally reserve such variable names for internal action code logic and not expose this format to the end user.
Is this variable something particular and specific to this API?